packages/asc-core/src/components/ButtonStyle/ButtonStyle.ts

```diff
@@ -25,7 +25,7 @@ const ButtonStyle = styled(ButtonBaseStyle)<Props>`
   & svg {
     width: 30px;
```

Comment:
Is 30px the standard for svg's in this theme? If so, can we add this to the theme config?

Reply:
is fixed width, no need for config